### ISSUE TITLE
Rails 6 - No collection problem

### ISSUE THREAD

**Issue by ThomasSertorio** [ForestAdmin/forest-rails#329]
## Expected behavior
Expecting ForestRails to "see" our collections in our `rails 6.0.0`.

## Actual behavior
Installation and setup look alright but forestadmin-schema.json stays empty. As a result we get a no collection error on the Forest UI.

## Failure Logs
No specific logs apart from the forestadmin-schema.json being empty:
```
  "collections": [],
  "segments": [],
  "actions": [],
  "meta": {
    "database_type": "postgresql",
    "framework_version": "6.0.0",
    "liana": "forest-rails",
    "liana_version": "3.3.0",
    "orm_version": "6.0.0"
  }
```

## Context
**Here is the context**: We 'd like to keep the "in app"  installation in rails rather than going full-microservice as recommended in the lastest documentation: in our specific case, losing "automatic" validation, enum, has_many relationships, model instance method is a no go. So we "fake" installed Forest through npm to get our environment secret. When I check the result of this setup, everything looks 👍  : collection are retrieved and exposed to the ForestAdmin UI. Then we use the `rails g forest_liana:install <environement_secret>` command with the retrieved environement secret to switch to the rails setup. When doing this on our `rails 6.0.0`, we get an empty `forestadmin-schema.json`.

We've been trying a lot of different things:
- When doing the exact same process ☝️ on a `rails 5.2`, everything is fine !
- Then when upgrading the app to a `rails 6.0` and restarting our server... The `forestadmin-schema.json` is "cleared" and we get back to a "no collection problem"
- We've checked credentials are ok: they're ok
- We've checked there was no CORS issue: there was not

* Package Version: 3.3.0
* Rails Version: 6.0.0
* Database Dialect: Postgresql
* Database Version: 9.3

**Comment by baptistetruchet** [comment]
I'm facing exactly the same issue. Everything is fine on `rails 5.2.3` but collections remain empty on `rails 6.0.0` (no error messages)

**Comment by arnaudbesnier** [comment]
Hi @ThomasSertorio @baptistetruchet It should be fixed with the liana v4.1.0.

Feel free to reopen this ticket if the issue persists.
